app/services/bulk_tagging/search.rb

```diff
@@ -27,7 +27,8 @@ def gds_response
         document_type: document_type,
         page: page,
         q: query,
-        fields: [:content_id, :document_type, :title, :base_path]
+        fields: [:content_id, :document_type, :title, :base_path],
+        search_in: 'title,base_path,details.internal_name'
```

Comment:
https://github.com/alphagov/publishing-api/blob/master/doc/api.md#get-v2content currently shows `search_in` as a comma-separated string whereas `fields` is an array, and `gds-api-adapters` passes all the params straight through so I don't think we can make this an array here.